docs/USAGE-GUIDE.md

```diff
@@ -251,6 +251,29 @@ typedef enum { S2N_STATUS_REQUEST_NONE, S2N_STATUS_REQUEST_OCSP } s2n_status_req
 status request an S2N_CLIENT should make during the handshake. The only
 supported status request type is OCSP, **S2N_STATUS_REQUEST_OCSP**.
 
+
+```c
+typedef enum { S2N_CERT_AUTH_NONE, S2N_CERT_AUTH_REQUIRED } s2n_cert_auth_type;
+```
+**s2n_cert_auth_type** is used to declare what type of client certificiate authentication to use.
+Currently the default for s2n is for neither the server side or the client side to use Client (aka Mutual) authentication.
+
```

Comment:
Totally agree with your comment. I was just trying to get at the messaging of the documentation. What do you think of something like  "`s2n_cert_auth_type` is used to declare what type of peer authentication to use. If this is not explicitly set, the default is to enable authentication of the server and to disable authentication of the client."?